mockery/outputter_test.go

```diff
@@ -0,0 +1,36 @@
+package mockery
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFilenameBare(t *testing.T) {
+	out := FileOutputStreamProvider{InPackage: false, TestOnly: false}
```

Comment:
I know testing a private object function is ugly but this is a move from testing a private function. I think it's a good intermediary step and not that far from being able to just test the object that is evolving.